Formula/graphite2.rb

```diff
@@ -28,7 +41,7 @@ def install
   test do
     resource("testfont").stage do
       shape = shell_output("#{bin}/gr2fonttest Simple-Graphite-Font.ttf 'abcde'")
-      assert_match /67.*36.*37.*38.*71/m, shape
+      assert_match(/67.*36.*37.*38.*71/m, shape)
```

Comment:
Heh, no worries 😄. I get the feeling we may have suppressed it in Rubocop but not much can be done about `ruby -wc` warnings beside ignoring them.